src/Microsoft.AspNet.JsonPatch/Helpers/PathHelpers.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.JsonPatch.Helpers
+{
+    internal static class PathHelpers
+    {
+        internal static CheckPathResult CheckPath(string pathToCheck)
```

Comment:
Would we ever do anything other than throw if the path is invalid? Wondering if this would be simpler as something like `string PathHelpers.NormalizePath(string)`